x-pack/test/observability_functional/apps/observability/alerts/index.ts

```diff
@@ -236,6 +245,65 @@ export default ({ getService }: FtrProviderContext) => {
           expect(await find.existsByCssSelector('[title="Rules and Connectors"]')).to.eql(true);
         });
       });
+
+      describe('Stat counters', () => {
```

Comment:
@claudiopro How about we move these tests in a separate file, similar to how we do for pagination, add_to_case etc? Initially we started adding all tests in this file, but later on we added a bit more structure and added new tests in their own files. Eventually we should refactor this big file and move things around. Do you want to create a ticket where we refactor existing tests and move them into separate files? I can also do it, as you wish!

Reply:
Sounds good!